.travis.yml

```diff
@@ -1,6 +1,36 @@
+sudo: false
 language: cpp
-script: autoreconf -i && ./configure && make check && ./test-program
+compiler:
+  - gcc
+  - clang
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - gcc-4.9
+      - g++-4.9
+      - clang
+before_install:
+  - if [ "$CXX" = "g++" ]; then 
+      export CXX="g++-4.9" CC="gcc-4.9";
+      export CXXFLAGS="-std=c++11";
+    elif [ "$CXX" = "clang++" ]; then
+      export CXXFLAGS="-std=c++11 -stdlib=libc++ -DGTEST_USE_OWN_TR1_TUPLE=1";
+    fi
```

Comment:
These are for the sake of Travis, although they're good clues on dependencies needed for building anywhere. However, I believe you have the `-DGTEST_USE_OWN_TR1_TUPLE=1` documented already for clang, and I think I added something about gcc-4.9 and c++11, but I'll double check and make sure it's all there somewhere.